django/db/models/lookups.py

```diff
@@ -349,8 +349,61 @@ def as_postgresql(self, compiler, connection):
         return "%s %s %s" % (lhs, self.postgres_operator, rhs), params
 
 
+class TupleLookupMixin:
```

Comment:
This is great stuff.

Any thoughts about moving it to `fields.composite` instead and use it to create `TupleExact`, `TupleGreaterThan`, etc to register them solely on `CompositePrimaryKey` and avoid the whole `is_tuple_lookup` logic?

Reply:
yes, of course, let's move it, I just thought this would be preferrable

Reply:
done 👍